core/arch/arm/kernel/boot.c

```diff
@@ -188,6 +188,29 @@ static void init_vfp_nsec(void)
 }
 #endif
 
+/*
+ * Check for supported Crypto Extensions (ARMv8 aarch32/aarch64)
+ * In case one of instructions is not supported false is returned.
+ */
+static bool check_cpuid_ce(void)
+{
+	uint32_t isar5 = read_isar5();
+
+	if (!(isar5 | ID_ISAR5_AES))
```

Comment:
`isar5 & ID_ISAR5_AES`?
How did you test this?

Reply:
oops, I messed up. thanks!

I was testing in QEMUv8 with CE support enabled (current default setup), however didn't bother to rebuild with crypto disabled (not even sure if it's possible to do that from configure script, maybe manually disabling `CONFIG_ARM_AES_BUILTIN` etc in source tree)